docs/README.md

```diff
@@ -177,7 +177,28 @@ with a rule like [`copy_to_bin`](https://docs.aspect.build/aspect-build/bazel-li
 rules_js automatically mirrors the `bin` field from the `package.json` file of your npm dependencies
 to a Starlark API you can load from in your BUILD file or macro.
 
-For example, if you depend on the `typescript` npm package, you write this in `BUILD`:
+The format of the generated starlark file is `@<npm external repo name>//<package name where npm binary is loaded>:<npm package name>/package_json.bzl`.
```

Comment:
Not quite right. The package segment `<package name where npm binary is loaded>` should be `<package name where npm package is linked>`. This is the same package of the package.json that defines the dependency on the npm package you're loading the bin from.

In the example below, the load statement would be

```
load("@npm//:typescript/package_json.bzl", typescript_bin = "bin")
```

since the `typescript` dep is coming from the root `package.json` and it is linked to the root package at  `//:node_modules/typescript`

Reply:
Interesting. I'll update the PR! 

Can you point to some code and/or docs so I can solidify my understanding of the way this works?

Reply:
There are scattered examples throughout the rules & examples repos. Here is a simple one that shows the load package being different than the use package:

https://github.com/aspect-build/rules_ts/blob/5ea2d011369bf371d2431871bd178d6893766392/examples/typecheck_only/BUILD.bazel#L9

corresponding to the typescript version from this package.json:

https://github.com/aspect-build/rules_ts/blob/5ea2d011369bf371d2431871bd178d6893766392/examples/package.json#L17

Reply:
Updated! Thanks for the pointers!